local-cli/runIOS/runIOS.js

```diff
@@ -135,7 +135,7 @@ function runOnDevice(selectedDevice, scheme, xcodeProject, configuration){
   });
 }
 
-function buildProject(xcodeProject, udid, scheme, configuration = 'Debug') {
+function buildProject(xcodeProject, udid, scheme, configuration = 'Debug', launchPackager = false) {
```

Comment:
What versions of node is the default syntax allowed in? Have you checked for regressions in older runtimes?